### ISSUE TITLE
Should there be a section in 1.3.1 on the differences between layout tables and data tables

### ISSUE THREAD

**Issue by ljoakley** [w3c/wcag#3524]
Although most tables created currently are data tables there are hundreds of layout tables that exist. The axe accessibility library checks and flags when either data tables and layout tables are built incorrectly. Axe checks that a layout table does not use headers, captions or summary.

**Comment by mbgower** [comment]
@ljoakley if you want to take a stab at drafting something, we can look at it.

I'd argue that the use of tables for layout has trended down significantly over the 15 years since WCAG 2.0 was released, and if this has never been considered necessary until now, it is probably not a high priority. So I would definitely advise not spending too much time on it.

**Comment by bruce-usab** [comment]
@ljoakley I think [F46](https://www.w3.org/WAI/WCAG22/Techniques/failures/F46.html) is close to what you are looking for, but maybe tweak it a bit?
> Failure of Success Criterion 1.3.1 due to using TH elements, caption elements, or non-empty summary attributes in layout tables

**Comment by fstrr** [comment]
F46 might need a bit of a tweak as the `summary` attribute isn't part of the HTML living standard. Related to that, it looks as though I'd forgotten to go through those documents and fix links, remove references to XHTML, etc.

**Comment by mbgower** [comment]
@ljoakley do you want to take a stab at drafting this?

**Comment by ljoakley** [comment]
Bruce has added a comment of where the differences between the types of tables are detailed.

**Comment by bruce-usab** [comment]
Closing since
> mbgower moved this from Fully reviewed to Done in WCAG 2.x 2 weeks ago

@ljoakley please draft a PR if you would like to reopen this issue separately from #3566 (and I see that you actively commenting on that one).
